src/ai/backend/client/cli/session/lifecycle.py

```diff
@@ -1060,6 +1060,14 @@ async def export_tracker(bgtask_id):
                                 "The operation has been cancelled in the middle. "
                                 "(This may be due to server shutdown.)",
                             )
+                        elif ev.event == "bgtask_partial_success" or ev.event == "bgtask_done":
+                            issues = data.get("issues")
+                            if issues:
+                                for issue in issues:
+                                    print_fail(f"Issue reported: {issue}")
+                                completion_msg_func = lambda: print_warn(
+                                    f"Task finished with {len(issues)} issues."
+                                )
```

Comment:
It looks like the description property can be defined in the event. Let’s apply it as is for now.